jina/types/document/__init__.py

```diff
@@ -571,6 +571,8 @@ def mime_type(self, value: str):
         :param value: the acceptable MIME type, raise ``ValueError`` when MIME type is not
                 recognizable.
         """
+        if value == 'audio/wav':
```

Comment:
I agree with ur comment, this is a big code smell.

Who is assigning these mime type in the example?

Reply:
I guess it's different browsers. Check this one. I also tried on firefox and there are no issues.
https://stackoverflow.com/questions/44891157/why-wav-format-doesnt-have-same-mimetype-in-different-browsers

Reply:
But the time lenght is not showing properly. lol
![image](https://user-images.githubusercontent.com/44033547/108636458-501b3500-7453-11eb-84e2-9f79ed23c9d0.png)

Reply:
I have seen that mime type accept an add_mapping or something like this, maybe is a better way to handle this.

Reply:
I changed the code to `mimetypes.add_type('audio/wav', '.wav')` in `@mime_type.setter`.Or I create a new function like miietype_loader() to add missing types. But for now I only see 'audio/wav' is missing.